cmd/cel-key/node_types.go

```diff
@@ -42,32 +43,27 @@ func ParseDirectoryFlags(cmd *cobra.Command) error {
 		return nil
 	}
 
-	nodeType := cmd.Flag(nodeDirKey).Value.String()
-	if nodeType == "" {
-		return errors.New("no node type provided")
+	nodeTypeStr := cmd.Flag(nodeDirKey).Value.String()
+	nodeType := nodemod.ParseType(nodeTypeStr)
+	if !nodeType.IsValid() {
+		return errors.New("no or invalid node type provided")
```

Comment:
Nit: could return err from ParseType to specify the exact case:
- no node type 
- invalid node type

Reply:
I agree, but its very low value to spend cycles on rn